R/get_attribution.R

```diff
@@ -98,40 +155,34 @@ get_attribution <- function(uuid = NULL, img = NULL, text = FALSE) {
     )
     # Add license title
     att$license_abbr <- licenses$abbr[which(licenses$links == att$license)]
-  }
-  # Format data
-  if (length(uuid) == 1 && text) {
     # Text output desired?
     if (text) {
-      att <- paste0("Silhouette was contributed by ",
+      txt <- paste0("Silhouette was contributed by ",
                     att$contributor, ", ",
```

Comment:
It looks like "contributors" are sometimes different from "attributors" (see https://www.phylopic.org/permalinks/33fa492ff63116d933c0c8edda57800e05e703694acda5ffe089e2cb0dd1db59). I think we want to return both of these pieces of information in the lists, but maybe only include attributors in the paragraph as is done at that permalink.